.github/workflows/pull_request.yml

```diff
@@ -192,6 +192,13 @@ jobs:
         working-directory: ./tests
         run: ./gradlew clean test -Dtags=longevity-docker
 
+      - name: upload longevity logs artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: injection-framework-logs
+          retention-days: 3
+          path: ./tests/build/corfu-longevity-app/correctness.log
+
```

Comment:
Is it possible to show the logs directly in the previous step? like what we have in unit tests, then we could have much longer retention days. I think compatibility-test also has this problem, might be related with Gradle setup.

Reply:
I think it's not necessary. 3 days is more than enough, you can just download the artifact. If the artifact got deleted you just restart a job. 
In that case, you don't need to pollute logs with huge output.

Reply:
It will be pretty problematic to print 260 mb of logs
![image](https://user-images.githubusercontent.com/314292/126524083-6b8442a0-1c61-4499-b887-61c94e8abbcc.png)

Reply:
Got it, make sense. Thanks!